steps/setup_docker_continue_generic.yml

```diff
@@ -8,35 +8,50 @@ hints:
 - Make sure you ran the \`docker-compose up\` command from the **project root directory**
   and that it finished running. This may take a few moments.
 startFlow:
+  if:
+    conditions:
+    - conditionId: user_has_feature_flag
+      params:
+        featureFlag: 'codespaces'
+    then:
+      do:
+      - actionId: finish_step
 trigger:
   type: user_message
   params:
     person: head-of-rd
   flowNode:
     if:
       conditions:
-      - conditionId: is_user_message_text_ready_to_continue
+      - conditionId: user_has_feature_flag
+        equals: false
+        params:
+          featureFlag: 'codespaces'
       then:
```

Comment:
You don't need this. Already handled in the startFlow

Reply:
Me :(

Reply:
you mean that if the user has it it will skip so no need to be defensive here and wrap in if?

Reply:
yep